src/microbe_stage/systems/MicrobeMovementSystem.cs

```diff
@@ -97,6 +95,18 @@ protected override void Update(float delta, in Entity entity)
             lookVector = Vector3.Forward;
         }
 
+        // Simplify turns to 90 degrees to keep consistant turning speed
+        if ((position.Rotation * Vector3.Forward).SignedAngleTo(lookVector, Vector3.Up)
```

Comment:
`(position.Rotation * Vector3.Forward).SignedAngleTo(lookVector, Vector3.Up)`

could be extracted as a variable to not have to calculate that number twice (I'm pointing this out as the movement system runs for each microbe on each game simulation update).